### PR TITLE
Adding HeadBob to characterMovement

### PR DESCRIPTION
**Pull request by Rafal2707**
FirstPersonController Script updated - added HeadBob with parameters to adjust it as we want

### CODE REVIEW

**Review by giorgioperri — APPROVED** [review]
This is brilliant! I've just corrected a couple typos in the code but will approve the PR. 

For future reference: make a copy of the scene and work on that. If you don't have to touch a scene, then it is better not to commit it! 

(I am keeping your editing because they represent the bobbing values you set to the player!).

Will wait for Felix's approval but this is fine with me!

**Review by flexomatico — APPROVED** [review]
Looking good!

**Pull request merged**
